gateway/1.4/modules/ROOT/pages/_partials/task-shared-storage-config.adoc

```diff
@@ -8,19 +8,19 @@ Production workflows should use https://redis.io/[Redis^], though defining it is
 You configure shared storage via a custom YAML file.
 
 The following steps demonstrate Redis-based storage support for Flex Gateway running in a Docker container, as a Linux Service,
-or as a Kubernetes Ingress controller.
+or as a Kubernetes cluster.
```

Comment:
what about OpenShift?

Reply:
Generally, a kubernetes cluster refers to the actual nodes and k8s control plane. I think it would be better to say that this is a deployment (a deployment in the k8s cluster)
```suggestion
or as a Kubernetes deployment.
```

Reply:
see Seba's comment below